styles.css

```diff
@@ -0,0 +1,94 @@
+body {
+  background-color: #707070;
+}
+
+@media (min-width: 481px) {
+  .header {
+    position: fixed;
+    float: left;
+    z-index: 3;
+    background-color: transparent;
+    color: #c0c0c0;
+    width: 100%;
+    height: 2.5em;
+    top: -8px;
+    left: 1em;
+  }
+  .buttons {
+    position: fixed;
+    float: right;
+    z-index: 2;
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+    align-items: center;
+    background-color: #000;
+    color: #c0c0c0;
+    width: 100%;
+    height: 3em;
+    top: 0px;
+    left: 0px;
+  }
+  .menubutton {
+    width: 8%;
+    min-width: 80px;
+    color: #FFF;
+  }
+}
+
+@media (max-width: 480px) {
+  body {
+    font-size: 1.4rem;
+  }
+  .header {
+    position: fixed;
+    z-index: 3;
+    background-color: #000;
+    color: #FFF;
+    width: 100%;
+    height: 2.5em;
+    margin: -20px auto 0px auto;
+    padding: 10px;
+    top: 0px;
+    left: 0px;
+  }
+  .buttons {
+    position: fixed;
+    z-index: 2;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-evenly;
+    align-items: center;
+    background-color: #000;
+    color: #c0c0c0;
+    width: 100%;
+    height: 3em;
+    bottom: 0px;
+    left: 0px;
+  }
+  .menubutton {
+    width: 8%;
+    min-width: 80px;
+    color: #FFF;
+  }
+}
+
+.content {
+  background-color: #c0c0c0;
+  border-radius: 15PX;
+  padding: 10px;
+  color: #000;
+  margin: 4em auto 0px auto;
+  width: 60%;
+}
+
+#image {
+  max-width: 100%;
+  display: block;
+  height: auto;
+  margin: 0 auto;
+}
+
+#img-caption {
+  color: #FFF;
+}
```

Comment:
Missing newline (I thought you just added this? I can't see it)

Reply:
I think I lost it on the autoformatting. Readding it now.